app/assets/stylesheets/styles/dashboard.scss

```diff
@@ -0,0 +1,22 @@
+// Place all the styles related to the dashboard controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+nav .nav-wrapper {
+  padding-right: 30px;
+  i {
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 3